app/models/post.rb

```diff
@@ -2,4 +2,14 @@ class Post < ApplicationRecord
   belongs_to :author, class_name: 'User', foreign_key: 'author_id'
   has_many :comments
   has_many :likes
+
```

Comment:
- [ ] Same here for the user, I recommend using : 
````ruby
belongs_to :author, class_name: 'User'
````

Reply:
Solved :+1: